conjur/controller/init_controller.py

```diff
@@ -42,22 +42,28 @@ def load(self):
         """
         Method that facilitates all method calls in this class
         """
+        if self.conjurrc_data.conjur_url is None:
+            self.get_conjur_server_url()
+        formatted_conjur_url = self.format_conjur_server_url()
+
         if self.ssl_verify is True:
-            fetched_certificate = self.get_server_certificate()
+            fetched_certificate = self.get_server_certificate(formatted_conjur_url)
             # For a uniform experience, regardless if the certificate is self-signed
             # or CA-signed, we will write the certificate on the machine
             self.write_certificate(fetched_certificate)
         else:
+            logging.warning('You chose to initialize the client in insecure mode. Reinitialize '
+                            'the client should you choose to communicate to the server securely')
```

Comment:
@InbalZilberman need your 👀  here. When we initialize the CLI we want to print a warning _once_ instead of an ugly
```
/Users/ssax/Documents/work/conjur-api-python3/venv/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'sigalmastercli.aim-dev.conjur.net'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
```

on each command. Example:

Previously:
```
conjur --insecure init
Enter the URL of your Conjur server (use HTTPS prefix): https://sigalmastercli.aim-dev.conjur.net
/Users/ssax/Documents/work/conjur-api-python3/venv/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'sigalmastercli.aim-dev.conjur.net'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
File /Users/ssax/.conjurrc exists. Overwrite? yes/no (Default: yes): y
```
now
```
conjur --insecure init
Enter the URL of your Conjur server (use HTTPS prefix): https://sigalmastercli.aim-dev.conjur.net
WARNING: You chose to initialize the client in insecure mode. Reinitialize the client should you choose to communicate to the server securely
...
File /Users/ssax/.conjurrc exists. Overwrite? yes/no (Default: yes): y
```

Reply:
@shulifink approval need here as well

Reply:
@InbalZilberman @shulifink reminder

Reply:
conjur --insecure init
Enter the URL of your Conjur server (use HTTPS prefix): https://sigalmastercli.aim-dev.conjur.net
WARNING: You chose to initialize the client in insecure mode. If you prefer to communicate with the server securely, you must reinitialize the client in secure mode.
File /Users/ssax/.conjurrc exists. Overwrite? yes/no (Default: yes): y

Reply:
BTW, when the user enters yes or no, do they have to type the whole word, or can they use y and n like you did?

Reply:
yes there are shortcuts so y/Y/YES/yes and the same for "no"

Reply:
updated